### DISCUSSION TITLE
No recordings

### DISCUSSION THREAD

**Discussion by alis-maroc** [mynaparrot/plugNmeet-server#120] [General]
Hi, Thanks first for this great work...
No recording found with moodle plugin from pugnmmet on my own server..

Thanks

**Comment by jibon57** [comment]
Hi,

Make sure recorder is running. You can check status:
`systemctl status plugnmeet-recorder`

**Reply by alis-maroc** [reply]
Yes running
● plugnmeet-recorder.service - plugNmeet recorder
     Loaded: loaded (/etc/systemd/system/plugnmeet-recorder.service; enabled; vendor preset: enabled)
     Active: active (exited) since Tue 2022-09-20 15:58:03 UTC; 57min ago

**Reply by jibon57** [reply]
Did you mean no recorder found or you did recording but those weren't listing in moodle?

**Reply by jibon57** [reply]
Please also mention the plugnmeet moodle plugin version as well as plugnmeet server.

**Comment by alis-maroc** [comment]
Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-48-generic x86_64)
plugnmeet 1.0.3 (2022051200)

**Reply by jibon57** [reply]
This is older version. Lot of changes since then. If you update plugNmeet server then will require to update plugin as well as.

**Comment by alis-maroc** [comment]
version  1.0.11 (2022091800) gives this error when create room on moodle:
Exception - syntax error, unexpected 'string' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST)

And for my error, on server there is recording files, but not shown on moodle

**Reply by jibon57** [reply]
You'll require to update your PHP version to at least 7.4 better 8.0

**Reply by alis-maroc** [reply]
You mean on moodle, but what if i cannot upgrade ?
i use debian with ispconfig panel...
is there solution?

**Comment by alis-maroc** [comment]
i found solution without update, i install version 1.0.10 and record is here in moodle room
Thanks

**Reply by jibon57** [reply]
Nice 👍

**Reply by jibon57** [reply]
I've pushed new version 1.1.0. Can you try that one?
